package.json

```diff
@@ -17,12 +17,13 @@
     "url": "https://github.com/feedhenry/fh-cluster/issues"
   },
   "devDependencies": {
+    "chai": "^3.4.1",
     "grunt": "^0.4.5",
     "grunt-fh-build": "^0.3.0",
     "istanbul": "^0.3.22",
     "mocha": "^2.3.3",
     "proxyquire": "^1.7.3",
-    "sinon": "^1.17.1"
+    "sinon": "^1.17.2"
```

Comment:
Based on https://github.com/feedhenry/product_releases/pull/35#discussion-diff-46668522, could you remove the version unspecifiers from here?